test/membrane/aac/filler_test.exs

```diff
@@ -69,19 +69,19 @@ defmodule Membrane.AAC.FillerTest do
 
       {_data, buffer_generator} = Source.output_from_buffers(buffers)
 
+      caps = %Membrane.AAC{
+        profile: :LC,
+        # Values samples_per_frame and sample_rate are set to constants
+        # that will result in Filler element state frame_duration field equal 1
+        samples_per_frame: 1,
+        sample_rate: Membrane.Time.second(),
+        channels: 1
+      }
+
       generator = fn state, size ->
```

Comment:
it seems that when passing caps to the source as option, we can use the default generator